### PR TITLE
Add XcodeGen support

### PR DESCRIPTION
**Pull request by lechuckcaptain**
This PR add a basic configuration for XcodeGen, in order to remove from git tracking the files under the following paths:

- `Example/SBTUITestTunnel.xcodeproj/`
- `Example/SBTUITestTunnel.xcworkspace/`

I also took the chance to add a basic Github Action workflow, building the Example app and running the UITests suites already available.

The proposed implementation should make easier the contributions to the project, and enable a faster (and safer) implementation of the following improvements:
- Replacing of storyboards
- Adding automatic code formatters

### CODE REVIEW

File: Scripts/build_lib.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        else
+            puts "XcodeBuild status: 🚨 FAILED"
+        end
+        return result
+    end 
+
+    def self.make_destination()
+        platform = "iOS Simulator"
+        device = available_simulators()
+        destination = "platform=#{platform},name=#{device}"
```
Comment (tcamin) [inline Scripts/build_lib.rb:40]: Would it make sense to print which device has been used to run tests?

**Comment by tcamin** [comment]
The [_Pods.xcodeproj](https://github.com/Subito-it/SBTUITestTunnel/blob/master/_Pods.xcodeproj) can be deleted as well

**Comment by tcamin** [comment]
Another thing that could be added (in a separate PR) is to verify the integration with SPM checking that the sample project builds and that a single UI test runs. We had issues with this in the past

File: Scripts/build_lib.rb (hunk 1 above)
Comment (lechuckcaptain) [reply Scripts/build_lib.rb:40]: Yep, I'll add that

**Review by tcamin — APPROVED** [review]

**Pull request merged**